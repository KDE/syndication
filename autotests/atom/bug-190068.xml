<?xml version="1.0" encoding="utf-8"?>
<feed version="0.3" xmlns="http://purl.org/atom/ns#">
  <entry>
    <content type="xhtml" xml:base="derp">
      <div xmlns="derp"></div>
    </content>
  </entry>
  <entry>
    <content type="xhtml" xml:base="derp">
      <div xmlns="derp"></div>
    </content>
  </entry>
</feed>
