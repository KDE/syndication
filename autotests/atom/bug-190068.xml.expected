# Feed begin ######################
# Item begin ######################
id: #hash:2f74e30cc6f88289000622edd20262bc#
content: #<div xmlns="derp" :base="derp"/>#
# Item end ########################
# Item begin ######################
id: #hash:2f74e30cc6f88289000622edd20262bc#
content: #<div xmlns="derp" :base="derp"/>#
# Item end ########################
# Feed end ########################

